Objects/obmalloc.c

```diff
@@ -1231,6 +1231,24 @@ _PyObject_XDecRefDelayed(PyObject *ptr)
 }
 #endif
 
+#ifdef Py_GIL_DISABLED
+void
+_PyObject_XSetRefDelayed(PyObject **ptr, PyObject *value)
+{
+    PyObject *old = *ptr;
+    FT_ATOMIC_STORE_PTR_RELEASE(*ptr, value);
+    if (old == _Py_NULL) {
```

Comment:
```suggestion
    if (old == NULL) {
```

`_Py_NULL` is only needed in headers